### ISSUE TITLE
WProofreader UI can't be opened if the tracking changes mode is toggled on

### ISSUE THREAD

**Issue by Mgsy** [WebSpellChecker/wproofreader-ckeditor5#27]
## Steps to reproduce

1.  Turn on the tracking changes mode.
2.  Open the WProofreader dropdown in the toolbar.
3.  Click on some dropdown item.

## Current result

Nothing happens.

## Notes

Basically, in CKEditor 5, the Track Changes feature requires introducing an integration, otherwise, the feature's commands are blocked. I can see [WProofreader uses commands](https://github.com/WebSpellChecker/wproofreader-ckeditor5/blob/master/src/wproofreaderediting.js#L29-L32) for displaying the UI and toggling spellchecking on/off. As it doesn't have a custom Track Changes integration, those commands are simply blocked in the tracking changes mode.

It seems that enabling commands in Track Changes solves this issue:

```javascript
const isTrackChangesLoaded = editor.plugins.has( 'TrackChanges' );

if ( isTrackChangesLoaded ) {
	const trackChangesEditing = editor.plugins.get( 'TrackChangesEditing' );
	const proofreaderCommands = [ 'WProofreaderToggle', 'WProofreaderSettings', 'WProofreaderDialog' ];
	proofreaderCommands.forEach( command => trackChangesEditing.enableCommand( command ) );
}
```

If you'd like to learn more about integrating custom features with Track Changes, [see the guide](https://ckeditor.com/docs/ckeditor5/latest/features/collaboration/track-changes/track-changes-custom-features.html) which explains this process in details.

**Comment by jshaptala** [comment]
@Mgsy, thanks a lot for creating this task and sharing the steps how we can fix that. We have already discussed it and it is scheduled for the nearest release. We will keep you posted.

**Comment by jalners** [comment]
This bug was fixed.
Fixed npm package is [2.0.5](https://www.npmjs.com/package/@webspellchecker/wproofreader-ckeditor5/v/2.0.5).

**Comment by Mgsy** [comment]
Great to hear that! Thanks for the information :)
